app/src/main/java/com/monkeyteam/chimpagne/ui/HomeScreen.kt

```diff
@@ -67,22 +67,6 @@ import com.monkeyteam.chimpagne.ui.utilities.PromptLogin
 import com.monkeyteam.chimpagne.viewmodels.AccountViewModel
 import com.monkeyteam.chimpagne.viewmodels.FindEventsViewModel
 
-class LocationViewModel(myContext: Context) {
-  private val fusedLocationClient = LocationServices.getFusedLocationProviderClient(myContext)
-
-  fun startLocationUpdates(
-      myContext: Context,
-      onLocationSuccess: (lat: Double, lng: Double) -> Unit
-  ) {
-    if (ActivityCompat.checkSelfPermission(myContext, Manifest.permission.ACCESS_FINE_LOCATION) ==
-        PackageManager.PERMISSION_GRANTED) {
-      fusedLocationClient.lastLocation.addOnSuccessListener { location ->
-        location?.let { onLocationSuccess(it.latitude, it.longitude) }
-      }
-    }
-  }
-}
-
 @SuppressLint("StateFlowValueCalledInComposition", "MissingPermission")
```

Comment:
Missing permissions ? for the feed ? You might explain why you include this @SuppressLint